x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/more_less_section.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { FunctionComponent, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFlexItem, EuiButtonEmpty } from '@elastic/eui';
+
+interface Props {
+  align?: 'left' | 'right';
+}
+
+export const MoreLessSection: FunctionComponent<Props> = ({ align = 'left', children }) => {
```

Comment:
I think this component can be replaced with `EuiAccordion`, WDYT?

Reply:
I am not a huge fan of the accordion and the animation in this case. I prefer the snappiness of an actual toggle

![accordion_vs_custom](https://user-images.githubusercontent.com/2854616/110104082-9bf8a300-7d9e-11eb-9b8a-5bc11b79fc21.gif)

Reply:
I see your reasoning here @sebelga, but then having 2 different animations make the UI inconsistent. 
Should we change _Advanced settings_ to the new component too? 
I also have concerns about built-in a11y features that handle this kind of interaction in accordion. There are attributes like `aria-controls`, `aria-expanded`, `role`, `aria-labelledby` that we need to implement in the new component as well.

Reply:
I think Yulia raises great points about accessibility. My advice would be to leverage EUI as much as possible, and then drive changes at the EUI level to support the behavior you want @sebelga. For example, ask for a `timing` or `animation` prop to be added to `EuiAccordion`. This will have the benefit of getting input from the design team, it will share the new behavior to all EUI consumers to reduce the potential for duplication, and it will ensure consistency throughout our UIs.

Reply:
Big +1 to using EUI components and requesting desired enhancements on their repo.

Reply:
An additional future consideration here... there is an existing _Advanced setting_ to 'Disable animations' globally. Imagine a future - with user settings - where an individual can toggle this as their personal preference.

Reply:
Good point about accessibility @yuliacech and leverage EUI effort on this, I will replace the component with the accordion 👍

Reply:
There is an issue with the `<EuiAccordion />` that clips the combox helptext after the first line. I've opened an issue for it (https://github.com/elastic/eui/issues/4623). Should we block this PR until the issue is solved?